src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -22,7 +23,15 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Release|AnyCPU'" />
   <ItemGroup  Condition="'$(TargetGroup)'!='net46'">
-    <Compile Include=".\**\*.cs" />
+    <Compile Include=".\**\*.cs"
+             Exclude=".\System\Runtime\Serialization\FormatterServices.cs;
+                      .\System\Runtime\Serialization\StreamingContext.cs"
+       />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netcore50'">
+    <!-- throws NotImplementedException -->
+    <Compile Include=".\System\Runtime\Serialization\FormatterServices.cs" />
+    <Compile Include=".\System\Runtime\Serialization\StreamingContext.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='net46'">
```

Comment:
If I build with /p:targetgroup=net46, I get an error thus: "The partial facade assembly's name, version, and public key token must exactly mat
ch the contract to be filled. Contract: Assembly(Name=System.Runtime.Serialization.Primitives, Version=4.2.0.0, Culture
=neutral, PublicKeyToken=b03f5f7f11d50a3a), Facade: Assembly(Name=System.Runtime.Serialization.Primitives, Version=4.1.
0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a) [C:\git\corefx.dan\src\System.Runtime.Serialization.Primitives\s
rc\System.Runtime.Serialization.Primitives.csproj]"